mobile/src/main/java/com/example/android/uamp/MediaNotificationManager.java

```diff
@@ -47,6 +50,10 @@
 public class MediaNotificationManager extends BroadcastReceiver {
     private static final String TAG = LogHelper.makeLogTag(MediaNotificationManager.class);
 
+    private static final String CHANNEL_ID = "MusicApp_Channel";
+    private static final String CHANNEL_NAME = "MusicApp Notification Channel";
+    private static final String CHANNEL_DESCRIPTION = "Notification used by MusicApp";
```

Comment:
As above, please add this to a string resource and reference the name as UAMP. :)